contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -35,8 +36,10 @@ contract PixelDevsUkraineDonation is
         uint256 indexed newValue
     );
     event ContractStateUpdated(bool indexed oldValue, bool indexed newValue);
+    event WithdrawWalletUpdated(address indexed oldValue, address indexed newValue);
 
-    constructor() ERC721("PixelDevsUkraineDonation", "PXLDEV-UKRAINE") {
+    constructor(address _withdrawWallet) ERC721("PixelDevsUkraineDonation", "PXLDEV-UKRAINE") {
+        withdrawWallet = _withdrawWallet;
```

Comment:
And if we do default this to Ukraine DAO, maybe that would replace constructor change?

Reply:
Agreed, with the default address added the constructor will be changed. I'll implement.

Reply:
I think it's fair that we remove the event emitter
```solidity
event WithdrawWalletUpdated(address indexed oldValue, address indexed newValue);
```
And the setter function
```solidity
function setWithdrawWallet(address _withdrawWallet) public onlyOwner {
        emit WithdrawWalletUpdated(withdrawWallet, _withdrawWallet);
        withdrawWallet = _withdrawWallet;
    }
```
Since this smart contract is specifically created to raise funds and donate into UkraineDAO treasury.

Thoughts?

Reply:
So while the UkraineDAO treasury address gets hard coded we can also remove the constructor argument